### PR TITLE
KFSPTS-14052 update email validation regular expression

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package edu.cornell.kfs.gl.service.impl.fixture;
+
+public enum EmailAdressTestValue {
```
Comment (nkimble) [inline src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: Both enum name and file name contain a misspelling. Should be EmailAddressTestValue.

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    THREE_PART_DOMAIN_GOOD("ww5@a.b.c.org", true),
+    ALPHANUMERIC_DOMAIN_DOT_ADDRESS_GOOD("something.else@a2.com", true),
+    ALPHANUMERIC_DOMAIN_UNDERSCORE_ADDRESS_GOOD("something_else@a2.com", true),
+    ALPHANUMERIC_DOMAIN_HYPHEN_ADDRESS_GOOD("something-else@a2.com", true),
+    UNDERSCORE_ADDRESS_GOOD("something_else@something.else.com", true),
+    HYPHEN_DOCMAIN_GOOD("something-else@et-tu.com", true),
```
Comment (nkimble) [inline src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: I think there is a typo here. Did you intend this to be HYPHEN_DOMAIN_GOOD ?

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    NO_ADDRESS_BAD("@a.b.c.org", false),
+    NO_DOMAIN_BAD("a", false),
+    SPACE_BAD(" @a", false),
+    HYPHEN_ADDRESS_BAD("\"-@a.org\"", false),
+    UNDERSCOREW_DOMAIN_BAD("someone@foo_bar.com", false);
```
Comment (nkimble) [inline src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: I think there is a typo here also. Should it be UNDERSCORE_DOMAIN_BAD ?

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    
+    public final String emailAdress;
+    public final boolean validAdress;
+
```
Comment (nkimble) [inline src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: Missing a "d" in both local variable names.

File: src/test/java/edu/cornell/kfs/kns/datadictionary/validation/fieldlevel/CuEmailAddressValidationPatternTest.java [hunk 5]
```diff
@@ -0,0 +1,80 @@
+package edu.cornell.kfs.kns.datadictionary.validation.fieldlevel;
+
+import org.junit.Test;
+import org.kuali.kfs.kns.datadictionary.validation.fieldlevel.EmailAddressValidationPattern;
+import org.powermock.api.mockito.PowerMockito;
+
+import edu.cornell.kfs.gl.service.impl.fixture.EmailAdressTestValue;
+
```
Comment (nkimble) [inline src/test/java/edu/cornell/kfs/kns/datadictionary/validation/fieldlevel/CuEmailAddressValidationPatternTest.java]: Test case will need to be adjusted for enum typos.

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java (hunk 1 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: OK

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java (hunk 2 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: fixed

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java (hunk 3 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: ok

File: src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java (hunk 4 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java]: ok

File: src/test/java/edu/cornell/kfs/kns/datadictionary/validation/fieldlevel/CuEmailAddressValidationPatternTest.java (hunk 5 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/kns/datadictionary/validation/fieldlevel/CuEmailAddressValidationPatternTest.java]: ok

**Comment by jhulslander** [comment]
I've made some changes, this is ready for review again

**Pull request merged**